app/javascript/mastodon/locales/fr.json

```diff
@@ -87,7 +87,7 @@
   "confirmations.mute.confirm": "Masquer",
```

Comment:
On `confirmations.domain_block.message` (line 86): *abonnés* instead of *suiveurs*.